Zend/tests/use_statement/aliasing_builtin_types.phpt

```diff
@@ -1,10 +0,0 @@
---TEST--
-Aliasing built-in types
---FILE--
-<?php
-
-use bool as A;
```

Comment:
see https://wiki.php.net/rfc/namespaced_names_as_token

`N\String` can be fixed if the removed/reverted check is done on whole imported token.

not allowing `use bool as A;` is probably a good idea

Reply:
This doesn't change anything, `bool` alone was also allowed. https://3v4l.org/QLicq